### DISCUSSION TITLE
Drawing Text In Graphics Mode

### DISCUSSION THREAD

**Discussion by prescott2m** [CosmosOS/Cosmos#1982] [Q&A]
Is There Any Possible Way To Draw Text in graphics mode?
i know `canvas.DrawChar` exists, but i am having trouble getting a font to use for it. please help!

**Comment by LuftKatzeBASIC** [comment]
https://wiki.osdev.org/VGA_Fonts#Displaying_a_character

**Comment by LuftKatzeBASIC** [comment]
https://github.com/spacerace/romfont/tree/master/font-headers copy font you like from *.h file to byte[]

**Comment by quajak** [comment]
The devkit has a DrawString function in the Canvas class.
